### ISSUE TITLE
aw rpa crashes with newest tty-command gem

### ISSUE THREAD

**Issue by Ingu** [thisismydesign/autowow#10]
https://rubygems.org/gems/tty-command updated to version 0.8.0 which breaks compatibility with this gem

```
Traceback (most recent call last):
        14: from /home/ingu/.rbenv/versions/2.5.1/bin/aw:23:in `<main>'
        13: from /home/ingu/.rbenv/versions/2.5.1/bin/aw:23:in `load'
        12: from /home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/autowow-0.9.0/bin/aw:5:in `<top (required)>'
        11: from /home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/base.rb:466:in `start'
        10: from /home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
         9: from /home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
         8: from /home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
         7: from /home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/autowow-0.9.0/lib/autowow/cli.rb:84:in `rubocop_parallel_autocorrect'
         6: from /home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/autowow-0.9.0/lib/autowow/features/gem.rb:35:in `rubocop_parallel_autocorrect'
         5: from /home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/autowow-0.9.0/lib/autowow/executor.rb:38:in `run!'
         4: from /home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/tty-command-0.8.0/lib/tty/command.rb:119:in `run!'
         3: from /home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/tty-command-0.8.0/lib/tty/command.rb:185:in `execute_command'
         2: from /home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/tty-command-0.8.0/lib/tty/command/process_runner.rb:64:in `run!'
         1: from /home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/autowow-0.9.0/lib/autowow/executor.rb:18:in `print_command_exit'
/home/ingu/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/tty-command-0.8.0/lib/tty/command/printers/pretty.rb:46:in `write': wrong number of arguments (given 1, expected 2..4) (ArgumentError)
```

**Comment by thisismydesign** [comment]
Thanks for reporting, will look into it.

**Comment by thisismydesign** [comment]
Fixed in [v0.9.1](https://github.com/thisismydesign/autowow/releases/tag/v0.9.1). I also improved dependency handling so the gem now works with more versions of `tty-command`. However I set an upper version limit which will need to be incremented manually. See #13 for more details and how this could be removed in the future.

Please note that I think there's another issue with `tty-command v0.8.0`: https://github.com/piotrmurach/tty-command/issues/46. So it might be worthwhile to stay on v0.7.0 for now.

**Comment by thisismydesign** [comment]
@Ingu FYI all issues regarding tty-command should be resolved in `v0.9.4`.
